### ISSUE TITLE
Can't claim nor deleted account...?

### ISSUE THREAD

**Issue by indrakaw** [keybase/keybase-issues#3798]
This is a bug.

I just signed up an account today, it's same as my GitHub's username.

I signed this up via Desktop software, and it's already has an account logged in. I went to 'Log in as another user' > 'Create an account', then I entered my username, after that my decice name and the account is created without any confirmation neither required a password to signed up.

I have a difficult to claim this username. 

I tried to recover it by entering the device from this option:
![image](https://user-images.githubusercontent.com/9590878/72681421-f761c600-3af5-11ea-8848-356c8b58b31d.png)

The first option, I don't think this can works because the device I'm on is it.

The second, it asking either having password. If I choose "Yes", it ask for an password - but I haven't entered it yet. Second does nothing.

I have no idea what to do.

**Comment by heronhaye** [comment]
Can you run `keybase log send` in the terminal and post your Log ID? Thanks.

**Comment by indrakaw** [comment]
Should I logged out first? I have an account logged in.

Or should I switch to another Windows account to get thing work?

**Comment by heronhaye** [comment]
Sorry, I don't understand your situation. Do you already have a Keybase account and are logged into it? Are you trying to make another account?

**Comment by indrakaw** [comment]
Yes, it was my alt and I was trying to make another account for professional reason.

**Comment by heronhaye** [comment]
What is the username of the account you're trying to create? Was it already made by someone else? Check keybase.io/accountname

**Comment by indrakaw** [comment]
It's keybase.io/indrakaw. I was trying to create this account. You can see the date it created, the timestamp is minutes before this issue posted.

**Comment by heronhaye** [comment]
Can you run `keybase log send`? It doesn't matter what account you're logged in as

**Comment by indrakaw** [comment]
Yes I can. 
![image](https://user-images.githubusercontent.com/9590878/73449019-ce162500-4394-11ea-8be7-50794f2b1b85.png)
It quite take a long time.

And sorry later replies, I was unable to get in touch with PC for a past week.

**Comment by indrakaw** [comment]
Keybase is latest version and I'm still having problem sending log.
```
λ keybase log send
This command will send recent keybase log entries to keybase.io
for debugging purposes only.

These logs don’t include your private keys, encrypted data or file names,
but they will include metadata Keybase normally can't read
(like file sizes and git repo names), for debugging purposes.

Continue sending logs to keybase.io? (type 'YES' to confirm): YES
Enter feedback (or <Enter> to send): This issue: https://github.com/keybase/keybase-issues/issues/3798
More feedback (or press <Enter> when done):
- ERROR API network error: Post https://api-0.core.keybaseapi.com/_/api/1.0/logdump/send.json: net/http: request canceled (Client.Timeout exceeded while awaiting headers)

λ keybase -v
keybase version 5.2.0-20200130014557+cf82db8320
```
